src/x86/sse2.rs

```diff
@@ -1762,6 +1762,16 @@ pub unsafe fn _mm_store_pd(mem_addr: *mut f64, a: f64x2) {
     *(mem_addr as *mut f64x2) = a;
 }
 
+/// Load a double-precision (64-bit) floating-point element from memory
+/// into both elements of returned vector.
+#[inline(always)]
+#[target_feature = "+sse2"]
+#[cfg_attr(test, assert_instr(movddup))]
```

Comment:
I've found that testing specific instructions for intrinsics that are basically just moves isn't too reliable. The specific mov variants seems to differ across targets, so I've just been leaving off tests for those.

Reply:
(And yeah, it looks like this is causing CI to fail on Linux.)

Reply:
Fixed.